src/meshapi/models.py

```diff
@@ -46,12 +47,17 @@ def __str__(self):
             return f"BIN {self.bin}"
         return f"MeshDB Building ID {self.id}"
 
+    def get_thumb(self):
+        if len(self.panoramas) >= 1:
+            return self.panoramas[0]
+        return None
+
 
 class Member(models.Model):
     name = models.TextField()
     primary_email_address = models.EmailField(null=True)
     stripe_email_address = models.EmailField(null=True, blank=True, default=None)
-    additional_email_addresses = ArrayField(EmailField(), null=True, blank=True, default=list)
+    additional_email_addresses = JSONFormArrayField(models.EmailField(), null=True, blank=True, default=list)
```

Comment:
What are the implications of this? Just that it's stored as a JSON array internally?

Or this makes the admin display nicer?

Reply:
Right, but why did you change the email addresses?

Reply:
so that it also gets the better widget.